change/@ni-nimble-components-db9e952a-a79d-4aa2-a8e9-b3455ee2b620.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update banner styling to align with visual design spec",
```

Comment:
Assuming we're happy with the approach in this PR, I'll update the two instances in azdo of buttons slotted within the banner to have `appearance-variant="primary"` before submitting this PR. I've verified that having the existing styling + `appearance-variant="primary"` doesn't have any adverse side-effects.

Reply:
Azdo PRs have been submitted